presto-cli/src/main/java/io/prestosql/cli/AlignedTablePrinter.java

```diff
@@ -186,6 +196,24 @@ static String formatHexDump(byte[] bytes)
         return HEX_BYTE_JOINER.join(createHexPairs(bytes));
     }
 
+    static String formatList(List<? extends Object> list)
+    {
+        ImmutableList.Builder<Object> formattedList = ImmutableList.builder();
+
+        list.forEach(value -> formattedList.add(formatValue(value)));
+
+        return formattedList.build().toString();
+    }
+
+    static String formatMap(Map<? extends Object, ? extends Object> map)
+    {
+        ImmutableMap.Builder<Object, Object> formattedMap = ImmutableMap.builder();
+
+        map.keySet().forEach(key -> formattedMap.put(key, formatValue(map.get(key))));
```

Comment:
should `key` be formatted as well?

Reply:
I guess so

Reply:
...  then you can't use Map to construct the final String (technically, the format function could return same String for two distinct keys)

Reply:
Yep, does it make sense right now?